src/lib/lifeCycle.ts

```diff
@@ -50,11 +50,12 @@ import {
 } from "./paths.js";
 import { confirm } from "./prompt.js";
 
-const ensureUserConfig = async (): Promise<UserConfig> => {
-  const userConfig = await initUserConfig();
+const ensureUserConfig = async (): Promise<void> => {
+  const maybeUserConfig = await initUserConfig();
```

Comment:
What does `initUserConfig` do? Does it _initialize_ config in a sense of creating it? Or does it read from the file?

If read, then why is it called `init`? Very confusing, every time I read it.